interview-questions.md

```diff
@@ -8,42 +8,49 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is object-oriented programming? How is it different than functional programming?
 
-Your answer:
+Your answer:Object-oriented programming is when coding is bases its foundations on coding around the object to make fuctional code. Such as creating a a method and bringing in object to perform in that method. Whereas fuctional programming is the process of producing code to have a specific behavior or perform a certain task. An example is creating a function and passing through arguments or data to perform the behavior or task it has been created for, like removing odd numbers in array where the array is the data inputted into a function and produces only even numbers.
 
-Researched answer:
+Researched answer: Object Oriented Programming are classes and objects. It is structured for simple reusable code. This is a good style of coding because it is can be dynamically used such as coding for movies: run time, title, year made. This code would be able to produce over all movies rather than being restricted to one. The four building blocks are Classes, Objects, Methods and Attributes.
 
 1. What is the difference between a Float and an Integer in Ruby?
 
-Your answer:
+Your answer:In Red Robin Ruby a Float and Integer are similar in the aspects of them both being numerical data and where they differ is Float will produce a decimal value and an Integer will give a a whole value with NO decimals.
```

Comment:
Exactly! What happens when we do something like 5 / 2? Would that produce a float or an integer?